### DISCUSSION TITLE
Tiff to pdf conversion /tiff and pdf merges

### DISCUSSION THREAD

**Discussion by laarani** [Hopding/pdf-lib#1470] [General]
Does pdf-lib supports the below 
1.Tiff to pdf conversions ?
2. Tiff + pdf merges ?

Please share a documentation on that ?

If tiff-> pdf conversion is not a supported feature, any roadmap plans in future for tiff conversion to pdf /tiff + pdf merges ?

Appreciate your response .
Lok

**Comment by JohanJimenex** [comment]
x2
